src/App/App.js

```diff
@@ -1,31 +1,22 @@
 import React from 'react';
 import { BrowserRouter, Route } from 'react-router-dom';
 
-
 import Header from '../Components/Header/header';
 import NavBar from '../Components/Navbar/nav';
-import Profile from '../Components/Profile/Profile';
-import Dialogs from '../Components/Dialogs/dialogs';
-import News from '../Components/News/news';
-import Music from '../Components/Music/music';
-import Settings from '../Components/Settings/settings';
-
-import route from '../route';
+import NavItem from '../Components/Navbar/NavItem/navItem';
 
 import './App.css';
 
 function App() {
+  const Items = Object.values(NavItem);
+  const NavItemBlock = Items.map((item) => (<Route key={item.id} path={item.path} component={item.component} />));
```

Comment:
Consider creating a separate function to hold the map handler and naming it with a "Route" in the name.
E.g const itemRoute = () => {...}

Reply:
I've added new itemRoute.jsx in my App folder